lbrynet/file_manager/EncryptedFileDownloader.py

```diff
@@ -28,50 +28,46 @@ def log_status(sd_hash, status):
         status_string = "finished"
     else:
         status_string = "unknown"
-    log.info("stream %s is %s", short_hash(sd_hash), status_string)
+    log.debug("stream %s is %s", short_hash(sd_hash), status_string)
 
 
 class ManagedEncryptedFileDownloader(EncryptedFileSaver):
     STATUS_RUNNING = "running"
     STATUS_STOPPED = "stopped"
     STATUS_FINISHED = "finished"
 
-    def __init__(self, rowid, stream_hash, peer_finder, rate_limiter,
-                 blob_manager, stream_info_manager, lbry_file_manager,
-                 payment_rate_manager, wallet, download_directory,
-                 file_name=None):
+    def __init__(self, rowid, stream_hash, peer_finder, rate_limiter, blob_manager,
+                 stream_info_manager, lbry_file_manager, payment_rate_manager, wallet,
+                 download_directory, file_name=None, sd_hash=None, key=None, stream_name=None,
+                 suggested_file_name=None):
         EncryptedFileSaver.__init__(self, stream_hash, peer_finder,
                                     rate_limiter, blob_manager,
                                     stream_info_manager,
                                     payment_rate_manager, wallet,
                                     download_directory,
                                     file_name)
-        self.sd_hash = None
         self.rowid = rowid
         self.lbry_file_manager = lbry_file_manager
         self._saving_status = False
+        self.key = key
+        self.stream_name = stream_name
+        self.suggested_file_name = suggested_file_name
+        self.sd_hash = sd_hash
 
```

Comment:
I think these variable need to be passed down to EncryptedFileSaver instead of being intialized here because they are managed there or in another base class CryptStreamDownloader .